# generated by patch-package 6.4.8 on 2022-05-11 12:19:40
#
# command:
#   npx patch-package react-dnd
#
# declared package:
#   react-dnd: 10.0.2
#
diff --git a/node_modules/react-dnd/lib/common/DndProvider.d.ts b/node_modules/react-dnd/lib/common/DndProvider.d.ts
index a5737a0..b0554fc 100644
--- a/node_modules/react-dnd/lib/common/DndProvider.d.ts
+++ b/node_modules/react-dnd/lib/common/DndProvider.d.ts
@@ -7,6 +7,7 @@ export declare type DndProviderProps<BackendContext, BackendOptions> = {
     context?: BackendContext;
     options?: BackendOptions;
     debugMode?: boolean;
+    children?: ReactNode;
 };
 /**
  * A React component that provides the React-DnD context
